mythril/laser/ethereum/svm.py

```diff
@@ -172,13 +182,7 @@ def _get_covered_instructions(self) -> int:
     def exec(self, create=False, track_gas=False) -> Union[List[GlobalState], None]:
         final_states = []
         for global_state in self.strategy:
-            if self.execution_timeout and not create:
-                if (
-                    self.time + timedelta(seconds=self.execution_timeout)
-                    <= datetime.now()
-                ):
-                    return final_states + [global_state] if track_gas else None
-            elif self.create_timeout and create:
+            if self.create_timeout and create:
```

Comment:
is having this nested if necessary, both of these can be combined right? It might reduce some nesting.